packages/hash/api/src/graphql/typeDefs/knowledge/page.typedef.ts

```diff
@@ -0,0 +1,232 @@
+import { gql } from "apollo-server-express";
+
+export const knowledgePageTypedef = gql`
+  type KnowledgePage implements KnowledgeEntity {
+    contents: [KnowledgeBlock!]!
+    archived: Boolean
+    summary: String
+    title: String!
+    icon: String
```

Comment:
I think now would be a good time to add documentation for these

Reply:
Addressed in 654d8f3d6e46776182332204311026319f9459ef